tests/prefetch_related/models.py

```diff
@@ -65,7 +65,12 @@ class BookWithYear(Book):
 
 
 class Bio(models.Model):
-    author = models.OneToOneField(Author, models.CASCADE)
+    author = models.OneToOneField(
+        Author,
+        models.CASCADE,
+        primary_key=True,
+        to_field='name',
+    )
```

Comment:
It would be worth checking if these field changes render other tests relying on the `Bio` model moot.